ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/management-api-models.j2

```diff
@@ -0,0 +1,18 @@
+{# Management API Models #}
+{% if management_api_models is arista.avd.defined %}
+!
+management api models
+{%     for provider in management_api_models.providers | arista.avd.natural_sort('name') %}
+   !
+   provider {{ provider.name }}
+{%         if provider.name is arista.avd.defined and provider.name | lower in ['smash', 'sysdb'] %}
```

Comment:
This will cause issues. In line7 you are using the var ```provider.name``` and then in line8 testing if that var is defined or not.

As you wanted to develop only fewer models for now, I suggested the following change.
```suggestion
{%         if provider.name is arista.avd.defined and provider.name | lower in ['smash', 'sysdb'] %}
   !
   provider {{ provider.name }}
```

Reply:
sure. I'll update this change. 

Also, should we keep the line 8 as follows 

`provider {{ provider.name | lower }}`

to make the intended config consistent with the EOS applied config, in case the user enters uppercase provider name?

Reply:
I vote for yes for keeping `lower`

Reply:
updated the template with the requested change.